### PR TITLE
Oberursel

### PR DESCRIPTION
**Pull request by telegnom**
Added Oberursel marktes.

### CODE REVIEW

**Comment by telegnom** [comment]
Is there any tag to specify the type of market, e.g. Farmer Market or Flea Market? I didn't find anything in the examples or contribution guide. :(

**Comment by johnjohndoe** [comment]
No not yet. Market classification is a pending feature, see #65.

**Comment by telegnom** [comment]
I'll add the flea markets to night, except the Auto-Show because it's not a market, it's more like a car fair sponsored by the city of Oberursel.

**Comment by johnjohndoe** [comment]
@telegnom Did you finish you changes? I cannot see any changes yet.

**Comment by telegnom** [comment]
Not yet, I'm sorry. I was short of time during the last days. I hope I'll have it done by the end of this week.

**Comment by johnjohndoe** [comment]
@telegnom Friendly reminder :bouquet:

**Comment by johnjohndoe** [comment]
Hey @telegnom, what happened to you :eyes: and the missing markets?

**Comment by telegnom** [comment]
Hi,
a lot of water flowed down the Rhine since I was here for the last time, but let's drop it. 

The flea markets 'Kleiner Bücherflohmarkt' and 'Flohmarkt in der Allee' have been added. Even with the field "opening_hours_undefined" in mind, I wasn't able to find a proper  way to describe when these markets take place. They are different dates each year, so it's hard to describe.

File: cities/oberursel.json [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+	{
+            "geometry": {
+                "coordinates": [
+                  8.5849,
+                  50.19947
+                ],
+                "type": "Point"
+            },
+            "properties": {
+                "title": "Flohmarkt in der Allee",
```
Comment (johnjohndoe) [inline cities/oberursel.json]: "Flohmarkt in der Allee" is contained two times.

**Comment by johnjohndoe** [comment]
@telegnom I rebased and squashed your commits into a single commit here: 47d171a92d3ef5c4e09a2801f8c924895905cae0. I also removed the duplicate market. Finally, I merged the your pull request here: f7dea4279f7db347d9b9495579c1e470906195a5.

Thanks a lot for your contribution and following up on the change requests.
Now it is the time to spread the word. Here are good starting points:
- [ ] Send a tweet `#woistmarkt` with a **screenshot** and a **link** to the website (please share the tweet link here)
- [ ] Try to convince the city of Oberursel to **link** to our website, see #38

**Comment by johnjohndoe** [comment]
https://twitter.com/tbsprs/status/884848448960385026

**Pull request closed without merging**